app/components/forms/wizard/basic-details-step.js

```diff
@@ -269,6 +276,44 @@ export default Component.extend(FormMixin, EventWizardMixin, {
             }
           ]
         },
+        ticketMinPrice: {
+          identifier : 'min_price',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Minimum price for donation tickets required')
+            },
+            {
+              type   : 'number',
+              prompt : this.l10n.t('Invalid number')
+            },
+            {
+              type   : 'integer[1..]',
+              prompt : this.l10n.t('Ticket price needs to be greater than zero')
+            },
+            {
+              type   : 'checkMaxMin',
+              prompt : this.l10n.t('Ticket min price should be lesser than max price')
+            }
+          ]
+        },
+        ticketMaxPrice: {
+          identifier : 'max_price',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Maximum price for donation tickets required')
+            },
+            {
+              type   : 'number',
+              prompt : this.l10n.t('Invalid number')
+            },
```

Comment:
```js
{
              type   : 'empty',
              prompt : this.l10n.t('Maximum price for donation tickets required')
            },
            {
              type   : 'number',
              prompt : this.l10n.t('Invalid number')
            },
```
Same here rather than `number` and `empty` , Just use `Integer` .

Reply:
@kushthedude I think that the empty field validator is required to indicate that it is necessary for the organizer to specify the max. price for a donation. On the other hand, I agree with the removal  of the number validator.